src/frontendHighlighterApp/index.js

```diff
@@ -891,9 +891,23 @@ class AccessibilityCheckerHighlight {
 	}
 }
 
-window.addEventListener( 'DOMContentLoaded', () => {
-	new AccessibilityCheckerHighlight();
-	if ( window.edacFrontendHighlighterApp?.userCanFix ) {
-		fixSettingsModalInit();
+// Some systems (Cloudflare Rocket Loader) defers scripts for performance but that can
+// cause some DOMContentLoaded events to be missed. This is flag tracks if it run so we
+// can retry at a latter event listener.
+let initHasRun = false;
```

Comment:
I'm curious if this can be simplified by using the once parameter and combining the listeners.

```js
const initHighlighter = () => {
    new AccessibilityCheckerHighlight();
    if (window.edacFrontendHighlighterApp?.userCanFix) {
        fixSettingsModalInit();
    }
};

// Combine listeners for DOMContentLoaded and load, using `once` to ensure the handler runs only once
['DOMContentLoaded', 'load'].forEach(event => {
    window.addEventListener(event, initHighlighter, { once: true });
});
```

Reply:
I used an array loop for the listener events but the `once` prop  for `addEventListener` is once per listener and not once across all of them :(